webapp/src/Form/Type/ProblemUploadType.php

```diff
@@ -28,6 +29,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'accept' => 'application/zip',
             ],
         ]);
+        $builder->add('delete_data_first', CheckboxType::class, [
+            'help' => 'If checked, all old data for the problem will be deleted, if any. Only useful for existing problems.',
```

Comment:
```suggestion
            'help' => 'If checked, any old data for the problem will be deleted. Only useful for existing problems.',
```

```suggestion
            'help' => 'Only useful for existing problems. If checked, any old data for the problem will be deleted.',
```
Second is just to know how it looks, it makes reading slightly easier as people directly know that 9/10 times this is not relevant for their usecase.

Reply:
I'm not sure if I like this

Reply:
Both are only suggestions, feel free to ignore.

Reply:
What about `If checked, old data will be deleted.`?

The second half "Only useful for existing problems" can be omitted if we only show the box for existing problems, right?